resources/js/Components/Form/DynamicFields.vue

```diff
@@ -57,8 +58,25 @@
                 this.$toast.error(this.$translations.tryLater, this.$translations.operationFiled);
             }
             this.loading = false;
-        },
-    }
+		},
+		methods: {
+        	populateFields(payload){
+        		let fieldValues = payload;
```

Comment:
any value that isn't mutated later in js should be const and not let

const fieldValues,
const newFields
const let
and later in the loop
const field